src/commands/base-command.ts

```diff
@@ -103,6 +105,14 @@ async function selectWorkspace(project: Project, filter?: string): Promise<strin
     log()
     log(chalk.cyan(`We've detected multiple sites inside your repository`))
 
+    if (isCI) {
+      throw new Error(
+        `Configure the site you want to work with and try again. Sites detected: ${(project.workspace?.packages || [])
```

Comment:
Could the message include some information about people can accomplish the task of configuring the site?